marshmallow_utils/schemas/identifier.py

```diff
@@ -112,10 +92,11 @@ def validate_identifier(self, data, **kwargs):
 
         if identifier:
             # at this point, `scheme` is set or validation failed earlier
-            validation_function = self.allowed_schemes[scheme]
+            validation_function = self.allowed_schemes[scheme]["validator"]
             if not validation_function(identifier):
+                scheme_label = self.allowed_schemes[scheme]["label"]
```

Comment:
minor: Should this be required when you identify an allowed scheme? Does it make sense to default to `scheme`?

Reply:
The idea was to make the whole configuration mandatory. Since the validator function *must* be there, one might as well just put a label (mainly because of translations). However, I do see that in some cases it might just be a redundant pain. The default will not allow translations but I added it :) Thanks!